src/website/src/pages/ec/page-structure/footer/docs/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  The footer is present on every page under ec.europa.eu. It provides
```

Comment:
```suggestion
  The footer is included in the last part of every page on a website.
```

Reply:
The original sentence seems clearer.
"on a website" is too vague what website? any website? the original sentence clearly mention "every page under ec.europa.eu"